README.md

```diff
@@ -63,6 +66,50 @@ $ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g
 root@656be163fe84:/app# BOLOS_SDK=$NANOS_SDK make scan-build
 ```
 
+## App testing
+
+With the `ledger-app-dev-tools` image, whether you are developing on macOS, Windows or Linux, you can quickly test your app with the [Speculos](https://github.com/LedgerHQ/speculos) emulator or the [Ragger](https://github.com/LedgerHQ/ragger) test framework.
+For examples of functional tests implemented with Ragger, you can have a look at the [app-boilerplate](https://github.com/LedgerHQ/app-boilerplate)
+
+First, run the `ledger-app-dev-tools` docker image. Depending on your platform, the command will change slightly :
+
+**Linux (Ubuntu)**
+
+```bash
+sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u):$(id -g) -v "/tmp/.X11-unix:/tmp/.X11-unix" -e DISPLAY=$DISPLAY ghcr.io/ledgerhq/ledger-app-builder/ledger-app-dev-tools:latest
+```
+
+**Windows (with PowerShell)**
+
+Assuming you already have a running X server like [VcXsrv](https://sourceforge.net/projects/vcxsrv/).
+
+```bash
+$fullPath = Resolve-Path .
+docker run --rm -ti -v "${fullPath}:/app" -e DISPLAY="host.docker.internal:0" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-dev-tools:latest
```

Comment:
remove `$fullPath = Resolve-Path` and replace `${fullPath}:/app` by `$(pwd):/app`

Reply:
pwd is an alias fo Get-Location in powershell, I suggest we use the native command so $(Get-Location) ?

Reply:
Agree. It's working well also.

Reply:
Done.